test/e2e/support/utils.js

```diff
@@ -103,10 +103,7 @@ export async function insertBlock( searchTerm ) {
 	// Waiting here is necessary because sometimes the inserter takes more time to
 	// render than Puppeteer takes to complete the 'click' action
 	await page.waitForSelector( '.editor-inserter__menu' );
-	await page.keyboard.type( searchTerm );
-	await page.keyboard.press( 'Tab' );
-	await page.keyboard.press( 'Tab' );
-	await page.keyboard.press( 'Enter' );
+	await page.click( `button[aria-label="${ searchTerm }"]` );
```

Comment:
Why this change? Won't it break with blocks that aren't featured (and hidden in the collapsed panels)?

Reply:
See https://github.com/WordPress/gutenberg/pull/6959#issuecomment-393894222.

Reply:
In my testing, this is the change that's causing the breakage in e2e tests. I understand the need to base the insertion on something less ambiguous than search. Maybe this should be a separate utility `insertBlockByLabel`

Reply:
Sure, but that separate utility will still fail? I'm unsure why `await page.click( `button[aria-label="${ searchTerm }"]` );` is failing elsewhere.